### PR TITLE
Return speaker and session data by id

### PR DESCRIPTION
**Pull request by aditya1702**
myscreenshot
I tested the functions using the following curl requests:
curl http://127.0.0.1:8001/api/v1/event/1/speakers/1
curl http://127.0.0.1:8001/api/v1/event/1/sessions/2

### CODE REVIEW

**Comment by aditya1702** [comment]
@leto could you have a look please :)

**Comment by championswimmer** [comment]
Using query with filter is more costly than doing get(id)

Given id is primary key, we are sure its unique. So why not use that.

File: open_event/views/views.py [hunk 1]
```diff
@@ -101,6 +109,14 @@ def get_speakers(event_id):
     """Returns all event's speakers"""
     speakers = Speaker.query.filter_by(event_id=event_id)
     return ObjectFormatter.get_json("speakers", speakers, request)
+
```
Comment (juslee) [inline open_event/views/views.py]: Codacy Issue found: [Trailing whitespace](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406637011/issues/source?bid=3233864&fileBranchId=3233882#l112)

**Comment by leto** [comment]
@aditya1702 this looks useful. Please switch to using `get(id)` like @championswimmer suggested and this should be good to merge

**Comment by aditya1702** [comment]
Ok @leto will do it! :)

**Comment by aditya1702** [comment]
@leto @championswimmer this is giving me error 
server1

I get the following error:
server

**Comment by championswimmer** [comment]
The returned object 'Speaker' isn't an array, so no 'filter_by' in it.

**Comment by aditya1702** [comment]
@championswimmer So this means we have to change the following line?
**return self.query.filter_by(**filters).order_by(column)**
Could you help me out here. I see that in your pull also you just added the view functions. So , how to pass the object instance to the Json function?

File: open_event/views/views.py [hunk 2]
```diff
@@ -66,6 +66,14 @@ def get_sessions(event_id):
     sessions = Session.query.filter_by(event_id=event_id, is_accepted=True)
     return ObjectFormatter.get_json("sessions", sessions, request)
 
+@app.route('/api/v1/event/<int:event_id>/sessions/<int:session_id>', methods=['GET'])
+@auto.doc()
+@cross_origin()
+def get_session_by_id(event_id,session_id):
```
Comment (juslee) [inline open_event/views/views.py]: Codacy Issue found: [Unused argument 'event_id'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2495617277/issues/source?bid=3233864&fileBranchId=3233882#l72)

File: open_event/views/views.py [hunk 3]
```diff
@@ -101,6 +109,14 @@ def get_speakers(event_id):
     """Returns all event's speakers"""
     speakers = Speaker.query.filter_by(event_id=event_id)
     return ObjectFormatter.get_json("speakers", speakers, request)
+    
+@app.route('/api/v1/event/<int:event_id>/speakers/<int:speaker_id>', methods=['GET'])
+@auto.doc()
+@cross_origin()
+def get_speaker_by_id(event_id,speaker_id):
```
Comment (juslee) [inline open_event/views/views.py]: Codacy Issue found: [Unused argument 'event_id'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2495617277/issues/source?bid=3233864&fileBranchId=3233882#l116)

**Comment by rafalkowalski** [comment]
@aditya1702  Please add tests

**Comment by aditya1702** [comment]
@rafalkowalski Ok. And why are the checks failing?

**Pull request merged**